### DISCUSSION TITLE
Plex Auto update

### DISCUSSION THREAD

**Discussion by zsmbrvr** [tteck/Proxmox#407] [Ideas]
Would it be possible to include https://github.com/mrworf/plexupdate into the Plex installer?

**Comment by tteck** [comment]
does the `apt update && apt upgrade -y` not work?

**Comment by zsmbrvr** [comment]
It probably does. The one I linked is a script that auto-updates plex, without having to long in to the server to do it.

**Comment by Nedlinin** [comment]
@tteck : Another note is that the plexupdate script allows people to use beta versions if they are PlexPass enabled.  The apt repo is just public releases.

**Reply by tteck** [reply]
Added to update script
